metabrainz/templates/index/funding.json

```
@@ -86,108 +86,116 @@
 
         "plans": [
             {
-            "guid": "one-time-15",
+            "guid": "one-time-custom",
             "status": "active",
-            "name": "Fifteen dollars one time donation",
-            "description": "A $15 one time end-user (non-commercial) donation.",
-            "amount": 15,
+            "name": "A custom one time donation",
+            "description": "A custom amount one time donation.",
+            "amount": 0,
             "currency": "USD",
             "frequency": "one-time",
             "channels": ["paypal", "bank", "credit-card"]
             },
             {
-            "guid": "one-time-25",
+            "guid": "one-time-one-month",
             "status": "active",
-            "name": "Twenty-five dollars one time donation",
-            "description": "A $25 one time end-user (non-commercial) donation.",
-            "amount": 25,
+            "name": "One time donation to cover one developer month",
+            "description": "$2500 one time support for 1 developer month",
+            "amount": 2500,
             "currency": "USD",
             "frequency": "one-time",
             "channels": ["paypal", "bank", "credit-card"]
             },
             {
-            "guid": "one-time-50",
+            "guid": "one-time-one-quarter",
             "status": "active",
-            "name": "Fifty dollars one time donation",
-            "description": "A $50 one time end-user (non-commercial) donation.",
-            "amount": 50,
+            "name": "One time donation to cover 3 developer months",
+            "description": "$7500 one time support for 3 developers months",
```

I suggest this way of writing for all entries: 
```suggestion
            "description": "$7500 one time support for 3 developer-months",
```